zigbee_weather_station_mcumgr/pm_static.yml

```diff
@@ -0,0 +1,127 @@
+EMPTY_0:
+  address: 0xec000
+  end_address: 0xef000
+  placement:
+    after:
+    - zboss_nvram
+  region: flash_primary
+  size: 0x3000
+app:
+  address: 0x10200
+  end_address: 0xe4000
+  region: flash_primary
+  size: 0xd3e00
+external_flash:
+  address: 0x120000
+  device: MX25R64
+  end_address: 0x800000
+  region: external_flash
+  size: 0x6e0000
+mcuboot:
+  address: 0x0
+  end_address: 0x10000
+  region: flash_primary
+  size: 0x10000
+mcuboot_pad:
+  address: 0x10000
+  end_address: 0x10200
+  region: flash_primary
+  size: 0x200
+mcuboot_primary:
+  address: 0x10000
+  end_address: 0xf0000
+  orig_span: &id001
+  - mcuboot_pad
+  - app
+  region: flash_primary
+  size: 0xe0000
+  span: *id001
+mcuboot_primary_1:
+  address: 0x0
+  device: flash_ctrl
+  end_address: 0x40000
+  region: ram_flash
+  size: 0x40000
+mcuboot_primary_app:
+  address: 0x10200
+  end_address: 0xf0000
+  orig_span: &id002
+  - app
+  region: flash_primary
+  size: 0xdfe00
+  span: *id002
+mcuboot_secondary:
+  address: 0x0
+  device: MX25R64
+  end_address: 0xe0000
+  region: external_flash
+  size: 0xe0000
+mcuboot_secondary_1:
+  address: 0xe0000
+  device: MX25R64
+  end_address: 0x120000
+  region: external_flash
+  size: 0x40000
+mcuboot_primary_2:
+  address: 0x120000
+  device: MX25R64
+  end_address: 0x160000
+  region: external_flash
+  size: 0x40000
```

Comment:
Just noticed that this is not in sync with the linker file here on in the one it was copied from